x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_results.tsx

```diff
@@ -77,6 +82,28 @@ const groupResultsOnMessage = i18n.translate(
 );
 const resultsGroupedOffId = 'aiopsLogRateAnalysisGroupingOff';
 const resultsGroupedOnId = 'aiopsLogRateAnalysisGroupingOn';
+const fieldFilterHelpText = i18n.translate('xpack.aiops.logRateAnalysis.page.fieldFilterHelpText', {
+  defaultMessage:
+    'Deselect non-relevant fields to remove them from groups and click the Apply button to rerun the grouping.  Use the search bar to filter the list, then select/deselect multiple fields with the actions below.',
+});
+const columnsFilterHelpText = i18n.translate(
+  'xpack.aiops.logRateAnalysis.page.columnsFilterHelpText',
+  {
+    defaultMessage: 'Deselect non-relevant columns to remove them from the table.',
```

Comment:
I'm not sure that any help text is needed for the 'Columns' control. But maybe it is useful to clarify how it's different to the 'Filter fields' control.

Reply:
I updated this to `Configure visible columns` - that seems like a more apt description. Thoughts?
72f120e9a39c579999d4b3aabcf548ff88d6fdf0

Reply:
`Configure visible columns` sounds good,